compiler/src/codegen/compcore.re

```diff
@@ -2983,6 +2975,23 @@ let compile_imports = (wasm_mod, env, {imports}) => {
     };
   };
 
+  let imports =
+    if (Env.is_normal_mode()) {
+      [
+        // HACK: Depend on runtime/string for the tests
```

Comment:
This was moved to only be included in normal mode, since the string module is now a Grain module and shouldn't depend on itself.

Reply:
No longer needed!